scripts/update-versions.py

```
@@ -71,15 +71,19 @@ def get_latest_module_release(repository: str, include_rc: bool = True) -> str:
     tags.sort(key=lambda v: (parse_version(v)[:3], parse_version(v)[3] or float('inf')))
     return tags[-1]
 
-def bump_bundle_if_needed(versions_data: Dict[str, Any], block: str) -> None:
-    """Bump the bundle version for a block if it hasn't already been incremented vs mainline."""
-    current = versions_data[block]["version"]
+def get_mainline_bundle_version(block: str) -> Optional[str]:
+    """Return the bundle version for a block on origin/mainline, or None if unavailable."""
     try:
         result = subprocess.check_output(['git', 'show', 'origin/mainline:versions.json'], text=True)
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)